### ISSUE TITLE
readthedocs build failing

### ISSUE THREAD

**Issue by TomAugspurger** [dask/dask-kubernetes#236]
https://readthedocs.org/projects/dask-kubernetes/builds/9806096/

```
Running Sphinx v1.8.5

Traceback (most recent call last):
  File "/home/docs/checkouts/readthedocs.org/user_builds/dask-kubernetes/envs/latest/lib/python3.7/site-packages/sphinx/config.py", line 368, in eval_config_file
    execfile_(filename, namespace)
  File "/home/docs/checkouts/readthedocs.org/user_builds/dask-kubernetes/envs/latest/lib/python3.7/site-packages/sphinx/util/pycompat.py", line 150, in execfile_
    exec_(code, _globals)
  File "/home/docs/checkouts/readthedocs.org/user_builds/dask-kubernetes/checkouts/latest/doc/source/conf.py", line 68, in <module>
    from dask_kubernetes import __version__
  File "/home/docs/checkouts/readthedocs.org/user_builds/dask-kubernetes/envs/latest/lib/python3.7/site-packages/dask_kubernetes-0.9.2+51.g61f0189-py3.7.egg/dask_kubernetes/__init__.py", line 3, in <module>
    from .core import KubeCluster
  File "/home/docs/checkouts/readthedocs.org/user_builds/dask-kubernetes/envs/latest/lib/python3.7/site-packages/dask_kubernetes-0.9.2+51.g61f0189-py3.7.egg/dask_kubernetes/core.py", line 30, in <module>
    from .objects import (
  File "/home/docs/checkouts/readthedocs.org/user_builds/dask-kubernetes/envs/latest/lib/python3.7/site-packages/dask_kubernetes-0.9.2+51.g61f0189-py3.7.egg/dask_kubernetes/objects.py", line 17, in <module>
    SERIALIZATION_API_CLIENT = client.ApiClient()
  File "/home/docs/checkouts/readthedocs.org/user_builds/dask-kubernetes/envs/latest/lib/python3.7/site-packages/kubernetes_asyncio-10.0.0-py3.7.egg/kubernetes_asyncio/client/api_client.py", line 72, in __init__
    self.rest_client = rest.RESTClientObject(configuration)
  File "/home/docs/checkouts/readthedocs.org/user_builds/dask-kubernetes/envs/latest/lib/python3.7/site-packages/kubernetes_asyncio-10.0.0-py3.7.egg/kubernetes_asyncio/client/rest.py", line 69, in __init__
    ssl_context=ssl_context
TypeError: __init__() got an unexpected keyword argument 'ssl_context'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/docs/checkouts/readthedocs.org/user_builds/dask-kubernetes/envs/latest/lib/python3.7/site-packages/sphinx/cmd/build.py", line 303, in build_main
    args.tags, args.verbosity, args.jobs, args.keep_going)
  File "/home/docs/checkouts/readthedocs.org/user_builds/dask-kubernetes/envs/latest/lib/python3.7/site-packages/sphinx/application.py", line 201, in __init__
    self.config = Config.read(self.confdir, confoverrides or {}, self.tags)
  File "/home/docs/checkouts/readthedocs.org/user_builds/dask-kubernetes/envs/latest/lib/python3.7/site-packages/sphinx/config.py", line 203, in read
    namespace = eval_config_file(filename, tags)
  File "/home/docs/checkouts/readthedocs.org/user_builds/dask-kubernetes/envs/latest/lib/python3.7/site-packages/sphinx/config.py", line 380, in eval_config_file
    raise ConfigError(msg % traceback.format_exc())
sphinx.errors.ConfigError: There is a programmable error in your configuration file:

Traceback (most recent call last):
  File "/home/docs/checkouts/readthedocs.org/user_builds/dask-kubernetes/envs/latest/lib/python3.7/site-packages/sphinx/config.py", line 368, in eval_config_file
    execfile_(filename, namespace)
  File "/home/docs/checkouts/readthedocs.org/user_builds/dask-kubernetes/envs/latest/lib/python3.7/site-packages/sphinx/util/pycompat.py", line 150, in execfile_
    exec_(code, _globals)
  File "/home/docs/checkouts/readthedocs.org/user_builds/dask-kubernetes/checkouts/latest/doc/source/conf.py", line 68, in <module>
    from dask_kubernetes import __version__
  File "/home/docs/checkouts/readthedocs.org/user_builds/dask-kubernetes/envs/latest/lib/python3.7/site-packages/dask_kubernetes-0.9.2+51.g61f0189-py3.7.egg/dask_kubernetes/__init__.py", line 3, in <module>
    from .core import KubeCluster
  File "/home/docs/checkouts/readthedocs.org/user_builds/dask-kubernetes/envs/latest/lib/python3.7/site-packages/dask_kubernetes-0.9.2+51.g61f0189-py3.7.egg/dask_kubernetes/core.py", line 30, in <module>
    from .objects import (
  File "/home/docs/checkouts/readthedocs.org/user_builds/dask-kubernetes/envs/latest/lib/python3.7/site-packages/dask_kubernetes-0.9.2+51.g61f0189-py3.7.egg/dask_kubernetes/objects.py", line 17, in <module>
    SERIALIZATION_API_CLIENT = client.ApiClient()
  File "/home/docs/checkouts/readthedocs.org/user_builds/dask-kubernetes/envs/latest/lib/python3.7/site-packages/kubernetes_asyncio-10.0.0-py3.7.egg/kubernetes_asyncio/client/api_client.py", line 72, in __init__
    self.rest_client = rest.RESTClientObject(configuration)
  File "/home/docs/checkouts/readthedocs.org/user_builds/dask-kubernetes/envs/latest/lib/python3.7/site-packages/kubernetes_asyncio-10.0.0-py3.7.egg/kubernetes_asyncio/client/rest.py", line 69, in __init__
    ssl_context=ssl_context
TypeError: __init__() got an unexpected keyword argument 'ssl_context'


Configuration error:
There is a programmable error in your configuration file:

Traceback (most recent call last):
  File "/home/docs/checkouts/readthedocs.org/user_builds/dask-kubernetes/envs/latest/lib/python3.7/site-packages/sphinx/config.py", line 368, in eval_config_file
    execfile_(filename, namespace)
  File "/home/docs/checkouts/readthedocs.org/user_builds/dask-kubernetes/envs/latest/lib/python3.7/site-packages/sphinx/util/pycompat.py", line 150, in execfile_
    exec_(code, _globals)
  File "/home/docs/checkouts/readthedocs.org/user_builds/dask-kubernetes/checkouts/latest/doc/source/conf.py", line 68, in <module>
    from dask_kubernetes import __version__
  File "/home/docs/checkouts/readthedocs.org/user_builds/dask-kubernetes/envs/latest/lib/python3.7/site-packages/dask_kubernetes-0.9.2+51.g61f0189-py3.7.egg/dask_kubernetes/__init__.py", line 3, in <module>
    from .core import KubeCluster
  File "/home/docs/checkouts/readthedocs.org/user_builds/dask-kubernetes/envs/latest/lib/python3.7/site-packages/dask_kubernetes-0.9.2+51.g61f0189-py3.7.egg/dask_kubernetes/core.py", line 30, in <module>
    from .objects import (
  File "/home/docs/checkouts/readthedocs.org/user_builds/dask-kubernetes/envs/latest/lib/python3.7/site-packages/dask_kubernetes-0.9.2+51.g61f0189-py3.7.egg/dask_kubernetes/objects.py", line 17, in <module>
    SERIALIZATION_API_CLIENT = client.ApiClient()
  File "/home/docs/checkouts/readthedocs.org/user_builds/dask-kubernetes/envs/latest/lib/python3.7/site-packages/kubernetes_asyncio-10.0.0-py3.7.egg/kubernetes_asyncio/client/api_client.py", line 72, in __init__
    self.rest_client = rest.RESTClientObject(configuration)
  File "/home/docs/checkouts/readthedocs.org/user_builds/dask-kubernetes/envs/latest/lib/python3.7/site-packages/kubernetes_asyncio-10.0.0-py3.7.egg/kubernetes_asyncio/client/rest.py", line 69, in __init__
    ssl_context=ssl_context
TypeError: __init__() got an unexpected keyword argument 'ssl_context'

Exception ignored in: <function RESTClientObject.__del__ at 0x7fe435ed3840>
Traceback (most recent call last):
  File "/home/docs/checkouts/readthedocs.org/user_builds/dask-kubernetes/envs/latest/lib/python3.7/site-packages/kubernetes_asyncio-10.0.0-py3.7.egg/kubernetes_asyncio/client/rest.py", line 84, in __del__
AttributeError: 'RESTClientObject' object has no attribute 'pool_manager'
````

**Comment by nre** [comment]
The docs build was failing because of https://github.com/tomplus/kubernetes_asyncio/issues/87 which was fixed in v11.0.0 (26 Feb).

May we have a new build of these docs? Thanks!

**Comment by TomAugspurger** [comment]
Thanks @nre. I think we're building again, but started another one to be sure: https://readthedocs.org/projects/dask-kubernetes/builds/
